docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -247,23 +247,12 @@ on the GitHub repo:
 
 - The Azure DCAP Client has a build dependency on version 17134 of the Windows 10 SDK.
    - This can be added via the Visual Studio Installer under Individual Components > Windows 10 SDK (10.0.17134.0).
-- Assuming the resulting .nupkg is put into the `C:\Azure-DCAP-Client` folder, it can be installed using:
+- Install Azure.DCAP.Windows (https://www.nuget.org/packages/Azure.DCAP.Windows/) client:
   ```cmd
-  nuget.exe install Microsoft.Azure.DCAP.Client -ExcludeVersion -Source "C:\Azure-DCAP-Client;nuget.org" -OutputDirectory C:\openenclave\prereqs\nuget
+  nuget.exe install Azure.DCAP.Windows -Version 0.0.1 -OutputDirectory C:\openenclave\prereqs\nuget
+  copy C:\openenclave\prereqs\nuget\Azure.DCAP.Windows.0.0.1\dll\* C:\Windows\System32
```

Comment:
Should we actually recommend copying it to `C:\Windows\System32`? Is that something that normal applications are expected to do? Or should we do something that affects the DLL search order, so that it picks up the dcap client in the `LoadLibraryEx` call?

Reply:
No, in the merge with `add_dcap_client_target.cmake` this is no longer necessary and removed.